src/main/studio/ui/SearchPanel.java

```diff
@@ -160,22 +162,37 @@ private void doSearch(SearchContext context, SearchAction action) {
                     result = SearchEngine.replaceAll(textArea, context);
                 }
             } catch (IndexOutOfBoundsException e) {
-                editorPane.setTemporaryStatus("Error during replacement: " + e.getMessage());
+                editorPane.setTemporaryStatus("Error during replacement: " + e.getMessage(), true);
                 return;
             }
         }
 
         String status;
+        boolean isAlert = false;
         if (! result.wasFound()) {
             status = "Nothing was found";
+            isAlert = true;
         } else if (result.getMarkedCount() > 0) {
             status = "Marked " + result.getMarkedCount() + " occurrence(s)";
         } else if (action == SearchAction.Find) {
-            status = "Selected the first occurrence";
+            DocumentRange range = result.getMatchRange();
+            int startOffset = range.getStartOffset();
+            try {
+                int line = textArea.getLineOfOffset(startOffset);
+                int charOffset = startOffset-textArea.getLineStartOffset(line);
+                status = String.format("Text found at %d:%d", 1+line, 1+charOffset);
+                if (context.getSearchForward() ? (startOffset < pos) : (startOffset > pos)) {
+                    status = status + ". Reached the end of the document, wrapping around.";
+                    isAlert = true;
```

Comment:
And i wound't tread this situation as a problem

Reply:
This is the very reason I made this PR in the first place, as it's hard to notice when the search wraps around if you are looking at the code and the NetBeans editor had an obvious red flash to alert you which I got used to. It is bad for UX if convenience features like this get lost through upgrades.

Reply:
I agree that it is good to have an information that we wrap around. My view is that it is not normal message rather than an alert. My view that an alert is a problem.

Reply:
Maybe this should use a different kind of highlight?